services/apps/search_sync_worker/src/bin/sync-organization.ts

```diff
@@ -0,0 +1,24 @@
+import { SQS_CONFIG } from '@/conf'
+import { getServiceLogger } from '@crowd/logging'
+import { SearchSyncWorkerEmitter, getSqsClient } from '@crowd/sqs'
+
+const log = getServiceLogger()
+
+const processArguments = process.argv.slice(2)
+
+if (processArguments.length !== 1) {
+  log.error('Expected 1 argument: organizationId')
+  process.exit(1)
+}
+
+const organizationId = processArguments[0]
+
+setImmediate(async () => {
+  const sqsClient = getSqsClient(SQS_CONFIG())
+  const emitter = new SearchSyncWorkerEmitter(sqsClient, log)
+  await emitter.init()
+
+  await emitter.triggerOrganizationSync('blabla', organizationId)
```

Comment:
Not sure why the value `blabla` is being hardcoded 🤔 shouldn’t it be an argument, so that it's useful when we run the script for a tenant?

Reply:
Yes to have a smaller delay before the sync happens - until we have REST API on the search-sync-worker we can only do async processing - and if someone changes something manually he needs to wait until queue message is processed until he sees the changes on the frontend.

Reply:
for the `blabla` it's not needed anymore because we aren't using `tenantId` anymore for message group id - but I kept it as an argument to the `triggerOrganizationSync` so that maybe someday we will need it again for some grouping.